### DISCUSSION TITLE
App Redesign

### DISCUSSION THREAD

**Discussion by alainm23** [elementary/tasks#399] [Ideas]
### Problem

The task app is useful, but I feel that the design could be improved.

### Proposal

- Redesign the task view to make it more compact.
- Redesign the date and time selection widget.
- Group incomplete and completed tasks.
- Redesign the way to add tasks.

**Comment by danirabbit** [comment]
Yeah I agree there’s a lot lacking here and the current way of doing things is messy and has lots of issues. I’d like to complete the GTK4 port first though since it’s almost done. I think it mostly needs maps stuff to be fixed
